ansible/roles/postgresql/templates/pgbouncer.ini.j2

```diff
@@ -9,7 +9,7 @@
 [databases]
 {% for db in postgresql_dbs %}
 {% set db_host = db.get('host', localsettings.PG_DATABASE_HOST) %}
-{% if db_host == inventory_hostname or db_host == hot_standby_master or is_monolith|bool %}
+{% if db_host == inventory_hostname or (hot_standby_master is defined and db_host == hot_standby_master) or is_monolith|bool %}
```

Comment:
i think @calellowitz already did this here https://github.com/dimagi/commcarehq-ansible/pull/704/files
already merged.